dashboard/app/models/levels/external.rb

```diff
@@ -33,7 +33,7 @@ class External < DSLDefined
 
   # Check if the level has a hand-written submit button. Once all submit buttons are removed from markdown, this can go away.
   def has_submit_button?
-    properties['markdown'].try(:include?, 'next-stage') && properties['markdown'].try(:include?, 'submitButton')
+    properties['markdown'].try(:include?, 'next-lesson') && properties['markdown'].try(:include?, 'submitButton')
```

Comment:
I'd like someone on foundations to check me on this, but I'm not sure that this PR is translation-safe. I didn't realize we had hand-written buttons in our lessons 🙈 but the translations will likely be overwritten after the next sync.

I think you could check for `next-stage` or `next-lesson` here and wait for translations to slowly catch up. I'm also having trouble figuring out what would happen on a level in this case so maybe it wouldn't be too bad

Reply:
Ya I knew this might be a tough one to ship with translations. Definitely looking for guidance on the best way to approach this.